app/views/mail.scala.html

```diff
@@ -0,0 +1,48 @@
+<table class="mailtable" style="width: 100%;">
+    <tr>
+        <td>
+            <table class="contenttable" style="width: 90%;margin: auto;">
+                <tr>
+                    <td class="logobox" style="text-align: center;"><img class="logo" src="https://media.licdn.com/media/AAEAAQAAAAAAAAKKAAAAJGYyN2NhODQ4LTQ5MTYtNGEwYy05NjA0LTAwOTNiMDNjNmNhOQ.png"/>
+                        <br>
+                    </td>
+                </tr>
+                <tr>
+                    <td>
+                        &nbsp;
+                    </td>
+                </tr>
+                <tr>
+                    <td class="titlebox" style="background-color: rgba(0, 117, 162, 0.4);padding: 20px;">
+                        <span class="titleboxtitle" style="font-family: Arial;font-size: 20px;font-weight: bold;color: #000;">Friday Lunch !!!</span>
+                        <br><p></p>
+                        <span class="titleboxtext" style="font-family: Arial;font-size: 14px;color: #000;">It's a gentle reminder to to fill your availablity <a href="http://lunatech-lunch.cleverapps.io/" style="font-size: 18px;font-weight: bold;">Lunatech-lunch</a>.
```

Comment:
You wrote: 'It's a gentle reminder to to fill your availablity'
It should be: 'This is a gentle reminder to fill in your availability'   :)